.github/workflows/devel_image.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Push Development Image
+on:
+  push:
+    branches:
+      - devel
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Log in to registry
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+      - name: Pre-pull image to warm build cache
+        run: |
+          docker pull ghcr.io/${{ github.repository_owner }}/awx_devel:${GITHUB_REF##*/}
```

Comment:
The other files use `${{ github.base_ref }}` but this one uses `${GITHUB_REF##*/}`, is there a specific reason for that?

Reply:
`github.base_ref` is only available for `pull_request` events. This runs as post-merge task (`push` event) and this was the only way I could find to get just the branch name.